dist/client/style.css

```diff
@@ -111,4 +111,5 @@ canvas {
 
 .infoSpan {
 	display: block;
+  margin: 0;
```

Comment:
`dist/client/style.css` has tabs and spaces mixed — one should be chosen and used consistently. (This should probably be a separate issue).

Reply:
Nice catch!